src/datachain/lib/hf.py

```diff
@@ -98,7 +98,7 @@ def process(self, split: str = ""):
         with tqdm(desc=desc, unit=" rows") as pbar:
             for row in ds:
                 output_dict = {}
-                if split:
+                if split and "split" in self.output_schema.model_fields:
```

Comment:
needed because we added a `extra` forbidden and by some other (and relevant) logic "split" doesn't exist in some cases in the `self.output_schema` model